test/pipes.txt

```diff
@@ -2,6 +2,8 @@
 ;;; RUN: bash -c '%(wasm-stats)s <(%(wat2wasm)s /dev/stdin <<<"(module)" -o /dev/stdout)'
```

Comment:
I think it just never got the tests run on it (with `-r`) since it was committed, so any run of `run-tests.py -r` is going to make these whitespace changes.